ios/storybooknative.xcodeproj/project.pbxproj

```diff
@@ -1085,6 +1106,13 @@
 				5C04BEA7D09C4DC4B05BB661 /* Roboto-MediumItalic.ttf in Resources */,
 				90151CF9607749A28BB60FDD /* Roboto-Regular.ttf in Resources */,
 				F9A904A78F284EE6989903AF /* orbit-icons.ttf in Resources */,
+				D88FDFB084E3435482711AB1 /* orbit-icons.ttf in Resources */,
```

Comment:
Do you think this is related to your recent tests fix update? I'm just wondering why all fonts were added to `xcode` project for a second time.

Reply:
I'll check it

Reply:
I am working on it